apps/sharing/__init__.py

```diff
@@ -118,7 +92,7 @@ def count_term(count):
                         '{0} posts on localservice3', count).format(count)
 
 
-SERVICES_LIST = [DIGG, FACEBOOK, DELICIOUS, MYSPACE, FRIENDFEED, TWITTER]
+SERVICES_LIST = [FACEBOOK, TWITTER, G+, REDDIT, TUMBLR]
```

Comment:
It needs to be the class names you created above.

So it should be `SERVICES_LIST = [FACEBOOK, TWITTER, Googleplus, Reddit, Tumblr]`

Actually, to be coherent with the naming for the previous services, it would be better if you could use all uppercase names for the three new classes (and also use those uppercase names on the above line).

Thanks!